ZeekPluginDynamic.cmake

```diff
@@ -324,7 +324,7 @@ function(bro_plugin_end_dynamic)
     add_custom_command(OUTPUT ${_dist_output}
             COMMAND ${BRO_PLUGIN_BRO_CMAKE}/zeek-plugin-create-package.sh ${_plugin_name_canon} ${_plugin_dist}
             WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-            DEPENDS ${_plugin_lib}
+            DEPENDS ${_plugin_lib} ${_plugin_scripts}
```

Comment:
This should be `_plugin_script`, to match the naming `ZeekPluginCommon.make`. However, looking at the rest of these cmake files I notice that we already use the plural for related variables, such as `BRO_PLUGIN_SCRIPTS`. So I propose we go the other way and leave this one as is, and instead change `ZeekPluginCommon.cmake` to use the plural. It's also a bit more intuitive to say `zeek_plugin_scripts` in CMakeLists.txt.

Reply:
ok, sorry, I fixed this issue in a clone build tree and messed up applying the fix after having to fork, fixed